integration_tests/bindc_05b.h

```diff
@@ -1,2 +1,7 @@
+#ifndef BINDC_05BH
+#define BINDC_05BH
```

Comment:
What if the user doesn't define these guard macros in their header file? Probably then using `@ccall(header="header_name")` twice will give errors. I feel https://github.com/lcompilers/lpython/issues/1613#issuecomment-1481903888 syntax is better.

Reply:
We need to ensure a header file is included exactly once.

Reply:
> What if the user doesn't define these guard macros in their header file?

C/CPP uses a `set` to store the headers, so that won't be an issue.https://github.com/lcompilers/lpython/blob/b28a44046c9f375ce6b8f39b00226cdb36e1ccfe/src/libasr/codegen/asr_to_c_cpp.h#L114

Reply:
There is no issue.